From ab436599c6eeb8d87dc73851d32bc668a0505229 Mon Sep 17 00:00:00 2001
From: Uri Mashiach <uri.mashiach@compulab.co.il>
Date: Wed, 29 Dec 2021 17:43:37 +0200
Subject: [PATCH 104/126] cpl-imx8m-mini: dts: remove unrelevant sections

The file cpl-imx8m-mini.dtsi should include information regarding the
UCM-iMX8M-Mini module.

Remove all the sections that are not relevant to the  UCM-iMX8M-Mini
module.

Signed-off-by: Uri Mashiach <uri.mashiach@compulab.co.il>
---
 arch/arm/dts/cpl-imx8m-mini.dtsi | 148 ---------------------------------------
 1 file changed, 148 deletions(-)

diff --git a/arch/arm/dts/cpl-imx8m-mini.dtsi b/arch/arm/dts/cpl-imx8m-mini.dtsi
index 921b657f95..248bdac51d 100644
--- a/arch/arm/dts/cpl-imx8m-mini.dtsi
+++ b/arch/arm/dts/cpl-imx8m-mini.dtsi
@@ -44,26 +44,6 @@
 			default-state = "on";
 		};
 	};
-
-	dsi_host: dsi-host {
-		compatible = "samsung,sec-mipi-dsi";
-		status = "okay";
-	};
-
-	ili9881c_panel {
-		compatible = "startek,ili9881c";
-		reset-gpio = <&pca9555 6 GPIO_ACTIVE_LOW>;
-		dsi-lanes = <4>;
-		panel-width-mm = <63>;
-		panel-height-mm = <115>;
-		status = "okay";
-
-		port {
-			ili9881c_from_dsim: endpoint {
-				remote-endpoint = <&ili9881c_to_dsim>;
-			};
-		};
-	};
 };
 
 &iomuxc {
@@ -73,7 +53,6 @@
 	cpl-imx8m-mini {
 		pinctrl_hog_1: hoggrp-1 {
 			fsl,pins = <
-				MX8MM_IOMUXC_GPIO1_IO08_GPIO1_IO8       0x19
 				MX8MM_IOMUXC_GPIO1_IO00_GPIO1_IO0       0x140
 				MX8MM_IOMUXC_GPIO1_IO15_GPIO1_IO15      0x140
 			>;
@@ -99,18 +78,6 @@
 			>;
 		};
 
-		pinctrl_flexspi0: flexspi0grp {
-			fsl,pins = <
-				MX8MM_IOMUXC_NAND_ALE_QSPI_A_SCLK 0x1c4
-				MX8MM_IOMUXC_NAND_CE0_B_QSPI_A_SS0_B 0x84
-
-				MX8MM_IOMUXC_NAND_DATA00_QSPI_A_DATA0 0x84
-				MX8MM_IOMUXC_NAND_DATA01_QSPI_A_DATA1 0x84
-				MX8MM_IOMUXC_NAND_DATA02_QSPI_A_DATA2 0x84
-				MX8MM_IOMUXC_NAND_DATA03_QSPI_A_DATA3 0x84
-			>;
-		};
-
 		pinctrl_i2c1: i2c1grp {
 			fsl,pins = <
 				MX8MM_IOMUXC_I2C1_SCL_I2C1_SCL			0x400001c3
@@ -145,59 +112,6 @@
 			>;
 		};
 
-
-		pinctrl_uart4: uart4grp {
-			fsl,pins = <
-				MX8MM_IOMUXC_ECSPI2_MISO_UART4_DCE_CTS_B 0x49
-				MX8MM_IOMUXC_ECSPI2_MOSI_UART4_DCE_TX    0x49
-				MX8MM_IOMUXC_ECSPI2_SS0_UART4_DCE_RTS_B  0x49
-				MX8MM_IOMUXC_ECSPI2_SCLK_UART4_DCE_RX    0x49
-			>;
-		};
-
-		pinctrl_usdhc2_gpio: usdhc2grpgpio {
-			fsl,pins = <
-				MX8MM_IOMUXC_SD2_CD_B_GPIO2_IO12	0x41
-				MX8MM_IOMUXC_SD2_WP_USDHC2_WP		0
-			>;
-		};
-
-		pinctrl_usdhc2: usdhc2grp {
-			fsl,pins = <
-				MX8MM_IOMUXC_SD2_CLK_USDHC2_CLK		0x190
-				MX8MM_IOMUXC_SD2_CMD_USDHC2_CMD		0x1d0
-				MX8MM_IOMUXC_SD2_DATA0_USDHC2_DATA0	0x1d0
-				MX8MM_IOMUXC_SD2_DATA1_USDHC2_DATA1	0x1d0
-				MX8MM_IOMUXC_SD2_DATA2_USDHC2_DATA2	0x1d0
-				MX8MM_IOMUXC_SD2_DATA3_USDHC2_DATA3	0x1d0
-				MX8MM_IOMUXC_GPIO1_IO04_USDHC2_VSELECT	0x1d0
-			>;
-		};
-
-		pinctrl_usdhc2_100mhz: usdhc2grp100mhz {
-			fsl,pins = <
-				MX8MM_IOMUXC_SD2_CLK_USDHC2_CLK		0x194
-				MX8MM_IOMUXC_SD2_CMD_USDHC2_CMD		0x1d4
-				MX8MM_IOMUXC_SD2_DATA0_USDHC2_DATA0	0x1d4
-				MX8MM_IOMUXC_SD2_DATA1_USDHC2_DATA1	0x1d4
-				MX8MM_IOMUXC_SD2_DATA2_USDHC2_DATA2	0x1d4
-				MX8MM_IOMUXC_SD2_DATA3_USDHC2_DATA3	0x1d4
-				MX8MM_IOMUXC_GPIO1_IO04_USDHC2_VSELECT	0x1d0
-			>;
-		};
-
-		pinctrl_usdhc2_200mhz: usdhc2grp200mhz {
-			fsl,pins = <
-				MX8MM_IOMUXC_SD2_CLK_USDHC2_CLK		0x196
-				MX8MM_IOMUXC_SD2_CMD_USDHC2_CMD		0x1d6
-				MX8MM_IOMUXC_SD2_DATA0_USDHC2_DATA0	0x1d6
-				MX8MM_IOMUXC_SD2_DATA1_USDHC2_DATA1	0x1d6
-				MX8MM_IOMUXC_SD2_DATA2_USDHC2_DATA2	0x1d6
-				MX8MM_IOMUXC_SD2_DATA3_USDHC2_DATA3	0x1d6
-				MX8MM_IOMUXC_GPIO1_IO04_USDHC2_VSELECT	0x1d0
-			>;
-		};
-
 		pinctrl_usdhc3: usdhc3grp {
 			fsl,pins = <
 				MX8MM_IOMUXC_NAND_WE_B_USDHC3_CLK		0x40000190
@@ -265,13 +179,6 @@
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_i2c1>;
 	status = "okay";
-
-        pca9555:pca9555@20 {
-                compatible = "nxp,pca9555";
-                gpio-controller;
-                #gpio-cells = <2>;
-                reg = <0x20>;
-        };
 };
 
 &i2c2 {
@@ -437,24 +344,6 @@
 	status = "disabled";
 };
 
-&lcdif {
-	display = <&display0>;
-	status = "okay";
-	display0: display@0 {
-		bits-per-pixel = <24>;
-		bus-width = <24>;
-	};
-};
-
-&mipi_dsi {
-	status = "okay";
-	port@1 {
-		ili9881c_to_dsim: endpoint {
-			remote-endpoint = <&ili9881c_from_dsim>;
-		};
-	};
-};
-
 &fec1 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_fec1>;
@@ -477,25 +366,6 @@
 	};
 };
 
-&uart3 { /* console */
-	pinctrl-names = "default";
-	pinctrl-0 = <&pinctrl_uart3>;
-	status = "okay";
-};
-
-&usdhc2 {
-	pinctrl-names = "default", "state_100mhz", "state_200mhz";
-	pinctrl-0 = <&pinctrl_usdhc2>, <&pinctrl_usdhc2_gpio>;
-	pinctrl-1 = <&pinctrl_usdhc2_100mhz>, <&pinctrl_usdhc2_gpio>;
-	pinctrl-2 = <&pinctrl_usdhc2_200mhz>, <&pinctrl_usdhc2_gpio>;
-	cd-gpios = <&gpio2 12 GPIO_ACTIVE_LOW>;
-	bus-width = <4>;
-	fsl,wp-controller;
-	vmmc-supply = <&reg_usdhc2_vmmc>;
-	no-1-8-v;
-	status = "okay";
-};
-
 &usdhc3 {
 	pinctrl-names = "default", "state_100mhz", "state_200mhz";
 	pinctrl-0 = <&pinctrl_usdhc3>;
@@ -517,11 +387,6 @@
 	arm-supply = <&buck2_reg>;
 };
 
-&usbotg1 {
-	dr_mode = "otg";
-	status = "okay";
-};
-
 &usbotg2 {
 	dr_mode = "host";
 	status = "okay";
@@ -533,16 +398,3 @@
 &cpu_crit0 {
 	temperature = <115000>;
 };
-
-&lcdif {
-	enable_polarity_low;
-	/delete-property/ assigned-clocks;
-	/delete-property/ assigned-clock-parents;
-	/delete-property/ assigned-clock-rates;
-};
-
-&mipi_dsi {
-	/delete-property/ assigned-clocks;
-	/delete-property/ assigned-clock-parents;
-	/delete-property/ assigned-clock-rates;
-};
-- 
2.11.0

